arch/arm/soc/st_stm32/stm32f0/soc.h

```diff
@@ -43,6 +43,14 @@
 #include <stm32f0xx_ll_system.h>
 #endif /* CONFIG_CLOCK_CONTROL_STM32_CUBE */
 
+#ifdef CONFIG_I2C
+#include <stm32f0xx_ll_i2c.h>
+#endif
```

Comment:
You may enable I2C in a separate commit.

Reply:
OK, done